spec/Section 6 -- Execution.md

```diff
@@ -143,6 +147,136 @@ ExecuteMutation(mutation, schema, variableValues, initialValue):
+  * Let {responseStream} be the result of running {MapSourceToResponseEvent(sourceStream, subscription, schema, variableValues)}
+  * Return {responseStream}.
+
+CreateSourceEventStream(subscription, schema, variableValues, initialValue):
+
+  * Let {subscriptionType} be the root Subscription type in {schema}.
+  * Assert: {subscriptionType} is an Object type.
+  * Let {selectionSet} be the top level Selection Set in {subscription}.
+  * Let {rootField} be the first top level field in {selectionSet}.
+  * Let {argumentValues} be the result of {CoerceArgumentValues(subscriptionType, rootField, variableValues)}.
+  * Let {fieldStream} be the result of running {ResolveFieldEventStream(subscriptionType, initialValue, rootField, argumentValues)}.
+  * Return {fieldStream}.
+
+ResolveFieldEventStream(subscriptionType, rootValue, fieldName, argumentValues):
+  * Let {resolver} be the internal function provided by {subscriptionType} for
+    determining the resolved value of a field named {fieldName}.
+  * Return the result of calling {resolver}, providing {rootValue} and {argumentValues}.
+
+#### Publish Stream
+
+Each event in the underlying event stream triggers execution of the subscription
+selection set.
+
+MapSourceToResponseEvent(sourceStream, subscription, schema, variableValues):
+
+  * Return a new event stream {responseStream} which yields events as follows:
+  * For each {event} on {sourceStream}:
+    * Let {response} be the result of running
+      {ExecuteSubscriptionEvent(subscription, schema, variableValues, event)}.
+    * Yield an event containing {response}.
+
+ExecuteSubscriptionEvent(subscription, schema, variableValues, initialValue):
+
+  * Let {subscriptionType} be the root Subscription type in {schema}.
+  * Assert: {subscriptionType} is an Object type.
+  * Let {selectionSet} be the top level Selection Set in {subscription}.
+  * Let {data} be the result of running
+    {ExecuteSelectionSet(selectionSet, subscriptionType, initialValue, variableValues)}
+    *normally* (allowing parallelization).
+  * Let {errors} be any *field errors* produced while executing the
+    selection set.
+  * Return an unordered map containing {data} and {errors}.
+
+Note: in large scale subscription systems, the {ExecuteSubscriptionEvent} and
+{Subscribe} algorithms may be run on separate services to maintain predictable
+scaling properties. See the section below on Supporting Subscriptions at Scale.
+
+#### Unsubscribe
+
+Unsubscribe cancels the Response Stream. This is also a good opportunity for the
+server to clean up the underlying event stream and any other resources used by
+the subscription. Here are some example cases in which to Unsubscribe: client
+no longer wishes to receive payloads for a subscription; the source event stream
+produced an error or naturally ended; the server encountered an error during
+ExecuteSubscriptionEvent.
+
+Unsubscribe()
+
+  * Cancel {responseStream}
```

Comment:
Since the server is the producer of the response stream, if we use "stop observing", we probably give the impression that we are describing client behavior, whereas the previous sections clearly describe server behavior.